### PR TITLE
P3 ppm doc

### PR DESCRIPTION
**Pull request by diana-macrometa**
This PR documents tasks for the PPM tab.

### CODE REVIEW

File: docs/photoniq/p3/manage-ppm-service.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Perform the following steps to create a new PPM job:
+
+1. Log in to your PhotonIQ Performance Proxy dashboard.
+2. Click **PPM**.
+3. Click **Show Jobs**.
+4. Click **New Job**.
+5. In the **URL** field, enter the URL you want PPM to analyze.
+6. In the **Cron Expression** field, you can accept the default cron expression to set the job to run every 30 minutes, or you can customize the expression.
+7. Select the **Device Type** that you want PPM to analyze the site for: **Desktop** or **Mobile**.
+8. (Optional) Enter values in the Advanced Settings:
```
Comment (diana-macrometa) [inline docs/photoniq/p3/manage-ppm-service.md:62]: These fields need explanation and definition.

Comment (dwivedianurag) [reply docs/photoniq/p3/manage-ppm-service.md:62]: in the viewport you can define resolutions height and resolution width for which you want the  ppm to run the default 1024*1366

File: docs/photoniq/p3/manage-ppm-service.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+   - **Viewport** - 
+     - **Width** - 
+     - **Height** - 
+   - **Headers** - Click **Add** to add additional headers.
```
Comment (dwivedianurag) [inline docs/photoniq/p3/manage-ppm-service.md]: Any headers you want to pass when using ppm service

**Review by dwivedianurag — APPROVED** [review]
ok

**Pull request merged**